packages/cli/src/events/events.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { Get, RestController } from '@/decorators';
+import { AuthenticatedRequest } from '@/requests';
+
+import { EventService } from './event.service';
+
+@RestController('/events')
+export class EventsController {
+	constructor(private readonly eventService: EventService) {}
+
+	@Get('/session-started')
+	track(req: AuthenticatedRequest) {
+		const pushRef = req.headers['push-ref'] as string;
```

Comment:
Every authenticated request has this push ref? If so let's reflect this in the type?

```ts
export type AuthenticatedRequest<
	RouteParams = {},
	ResponseBody = {},
	RequestBody = {},
	RequestQuery = {},
> = Omit<APIRequest<RouteParams, ResponseBody, RequestBody, RequestQuery>, 'user' | 'cookies'> & {
	user: User;
	cookies: Record<string, string | undefined>;
	headers: express.Request['headers'] & {
		'push-ref': string;
	};
};
```